src/frontend/Player/createVideojsPlayer.spec.tsx

```diff
@@ -1,15 +1,15 @@
 import { render } from '@testing-library/react';
 import React from 'react';
 
-import VideoPlayer from '../components/VideoPlayer';
+import VideoPlayer from 'components/VideoPlayer';
```

Comment:
Is sorting imports alphabetically on purpose and should be consider as a new rule ? (are you eventually using a tslint rule ?)

Reply:
with absolute path yes it makes sense to add a new rule.

Reply:
And it should be on purpose yes